src/3.0/validate/validate.ts

```diff
@@ -38,44 +40,6 @@ const isValidRFC3986 = (str: any) => {
   return rfc3986.test(str);
 };
 
-const preloadedContextList = [
-  "https://www.w3.org/2018/credentials/v1",
-  "https://www.w3.org/2018/credentials/examples/v1",
-  "https://schemata.openattestation.com/com/openattestation/1.0/DrivingLicenceCredential.json",
-  "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
-  "https://schemata.openattestation.com/com/openattestation/1.0/CustomContext.json",
-];
-const contexts: Map<string, Promise<any>> = new Map();
-const nodeDocumentLoader = documentLoaders.xhr ? documentLoaders.xhr() : documentLoaders.node();
-let preload = true;
-
-const documentLoader = async (url: string) => {
-  if (preload) {
-    preload = false;
-    for (const url of preloadedContextList) {
-      contexts.set(
-        url,
-        fetch(url, { headers: { accept: "application/json" } }).then((res: any) => res.json())
-      );
-    }
-  }
-  if (contexts.get(url)) {
-    const promise = contexts.get(url);
-    return {
-      contextUrl: undefined, // this is for a context via a link header
-      document: await promise, // this is the actual document that was loaded
-      documentUrl: url, // this is the actual context URL after redirects
-    };
-  } else {
-    const promise = nodeDocumentLoader(url);
-    contexts.set(
-      url,
-      promise.then(({ document }) => document)
```

Comment:
i noticed that for the preloadedUrls, the object return from the fetch request is an object of `JsonLdObj`, but for other urls, you get an object of type `RemoteDocument`

Example of type `RemoteDocument`
```
export type JsonLd = JsonLdObj|JsonLdArray;

export interface RemoteDocument {
    contextUrl?: Url;
    documentUrl: Url;
    document: JsonLd;
}
```